chain/client/src/metrics.rs

```diff
@@ -27,6 +27,23 @@ pub(crate) static IS_VALIDATOR: Lazy<IntGauge> = Lazy::new(|| {
         .unwrap()
 });
 
+pub(crate) static IS_BLOCK_PRODUCER: Lazy<IntGauge> = Lazy::new(|| {
+    try_create_int_gauge(
+        "near_is_block_producer",
+        "Bool to denote if the node is currently a block producer",
+    )
+    .unwrap()
+});
+
+pub(crate) static IS_CHUNK_PRODUCER_FOR_SHARD: Lazy<IntGaugeVec> = Lazy::new(|| {
+    try_create_int_gauge_vec(
+        "near_is_chunk_producer_for_shard",
+        "Number of blocks expected to be produced by a validator",
```

Comment:
Is that so? :)

Reply:
and please clarify that it's for the current epoch.

Reply:
Copy-pasta